judge/judgedaemon.main.php

```diff
@@ -220,7 +220,10 @@ function rest_encode_file(string $file, $sizelimit = true) : string
     return base64_encode(dj_file_get_contents($file, $maxsize));
 }
 
-$waittime = 5;
+// Both constants in miliseconds
+const INITIAL_WAITTIME =  100;
```

Comment:
consider adding the suffix `_MS` to the variable name to make it easier to see the unit